### ISSUE TITLE
Project Meeting (Weekly) - ROW FY22 Fee Updates

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#6203]
**Why:** Meeting weekly with ROW supervisors to work on fee updates
**When:** Weekly on Fridays @10am, until 10/1/21
**Who:** Tracy, Ryan, Paloma, Darren, Jim, Peggy

**Comment by TracyLinder** [comment]
### 6/4/21 Action Items

- [ ] Merge Ryan's spreadsheet into mine
- [ ] Schedule scoping for EX & TURP updates, with Paloma, Izzy, Kim
- [ ] Draft AC for 6300 (In-process TCP)
- [ ] Schedule review of AIMS spreadsheet with Paloma, Peggy.
- [ ] (Paloma) Send Peggy portal documentation

**Comment by TracyLinder** [comment]
### 6/11/21 Updates 
- #6306 scoped TURP & EX fee logic updates
- Technical criteria on Add TCP Review Fee to SP, GP #6300
    - Next meeting on 6/14
- Meeting: Cost of Service: Review AIMS Spreadsheet #6307

**Comment by TracyLinder** [comment]
# 6/25

### Cost of Service
- Will the partner departments know the finance info at time of application? 
- SME's for partner departments is our biggest blocker right now
- We need to decide at some point if this data dump idea is possible; we need partner department feedback before we can know. We are having issues finding the right people to talk to. 

**Partner Departments:**
- AE
    - Dispatch Teams: Lucas Haydon, Steven Paul Peterson, Cleveland Allsup
- ATD
    - Amica's Team
    - Signals (Lauren S.)
- AWU
    - Dispatch group is applying for permits. Or it's the contractor. 
    - Enrique Coranado , James Randig, Leslie Jansen, Sandra Rogerio
- CPO
    -  Jessica Davis, Greg Pepper
- Project Connect
    - Currently hiring; Jen Golech 
- PWD
    - SSPD: John Eastman, or Javier 
    - SBO: Javier Martinez 
    - CIP: Ask John Eastman
- Watershed
    - Drainage Maintenance Crew: Roxanne Jackson

If you are a private entity applying for a city department, you are not currently charged. This would stay the same, and we'd back charge the departments 

### TCP In-Process Review Fee
- On the SP folder: Technical criteria is ready for review by ATD & DSD
- On the GP folder: Tracy is still trying to synthesize information and propose a technical solution


### Action Items 

- [ ] Tracy - Create DTS issues for TURP & EX enhancements
- [ ] Tracy - schedule discovery w/ ATD permit groups (Lauren, Amica, Peggy, Paloma)
- [ ] Meeting w/ AE already scheduled
- [ ] Tracy - Figure out GP TCP fee solution

**Comment by TracyLinder** [comment]
### 7/2/21

- Council make accept a recommendation to allow PSM fees to be waived. If so, we'll need a new fee waiver/adjustment type. We're not sure when we'll know more about this. 
- Tracy is stuck on the GP TCP fee solution. It's complex. 
**Cost of Service:**
We met with AE & AWU. We think AE can implement the proposed solution. AWU may not; they may have to figure out how to pay us outside of the solution we're proposing. 



### Action Items

- [ ] Tracy - Schedule time early next week to sync w/ peggy on cost of service things
- [ ] Tracy - Schedule time w Laura R to discuss the GP TCP fee
- [ ] Tracy - Create issues for ROW Usage fee updates on all permits types where applicable

**Comment by TracyLinder** [comment]
![Screen Shot 2021-07-06 at 4.10.06 PM.png](https://images.zenhubusercontent.com/5cca1e522b8dfc0ee14dedf5/9dcd84fc-5829-4605-b605-263396f0ec92)

**Comment by TracyLinder** [comment]
## 7/9/21

### Updates / Agenda
- TCP review fee will need to be invoiced on the GF folder for all TCP's. 
    - See #6300
- How will partner departments pay the TCP Fee? 
- Look at all issues in all pipelines
- Will & how will 'Secondary' permits pay TURP & EX permit extension fee?

**Comment by TracyLinder** [comment]
### 7/12 sync w/ Peggy

Austin Energy will be able to input the finance data at time of application
Austin Water is meeting internally to discuss how they'll progress

Need to meet with:

Public Works: John Eastman, Javier Martinez, Roxanne Cook

Watershed: Drainage Maintenance Crew: Roxanne Jackson

CPO: Jessica Davis, Greg Pepper

Project Connect: Jen Golech


Some departments will need to capture data on our permits that will help them pay our permit fees. 
AE: Work order #, Permit #, FDU, Sub-project # if exists

**Comment by TracyLinder** [comment]
I sent initial invites to all partner departments today.

**Comment by TracyLinder** [comment]
## 7/16

### Updates
- TCP Fee on GP & SP
- Cost of Service
- Review open issues


### Action Items

**Comment by TracyLinder** [comment]
### 7/23

**Agenda**
- Should #6279 be included in this projejct?

**Comment by TracyLinder** [comment]
## 8/6/21





### Action Items
- (New issue) New info - Work order #
- (New issue) FDU fields
    - Fund, Department, Unit, Object
    - Each limit to 4 numerical characters
